docs/organizations/verify-user-permissions.mdx

```diff
@@ -11,21 +11,11 @@ description: A collection of utility functions and components in order to allow
 
 Clerk provides two kinds of helpers for authorization:
 
-- **`protect` helpers** are opinionated and will block additional processing for unauthorized users, either by not rendering children or by throwing errors. The [`<Protect>` component](/docs/components/protect) and the experimental `auth().protect()` method fall into this category.
-- **`has` helpers** are more composable and flexible. They allow you to check and see what permissions a user _has_. You can decide what to do if the user lacks the permission.
+- `protect()` helpers are opinionated and will block additional processing for unauthorized users, either by not rendering children or by throwing errors. The [`<Protect>` component](/docs/components/protect) and the experimental [`auth().protect()`](/docs/references/nextjs/auth#protect) method fall into this category.
+- [`has()` helpers](/docs/references/nextjs/authentication-object#has) are more composable and flexible. They allow you to check and see what permissions a user *has*. You can decide what to do if the user lacks the permission.
```

Comment:
A couple questions:

1. Why do we refer to them as "kinds of helpers" in this section, instead of just "helpers"?
2. Is there more than one `has()`? If not, maybe we should refer to it as the `has()` helper, singular?

Reply:
I didn't write the original doc, so good catch! I will update the copy to remove "kinds of helpers".
and I agree about the singular usage

Reply:
So I didn't originally write this doc, but I think their impression was that they wanted to address an actual helper, and then a "helper" component. because we have protect() and we also have <Protect />
I am definitely open to changing up the wording here. Do you want to give it a shot?